azure-pipelines-dev.yml

```diff
@@ -149,9 +148,9 @@ stages:
     - job: unused_import_checker
       steps:
       - script: |
-          pip install $(grep -E '^(flake8)' reqs/requirements-dev-contrib.txt)
+          pip install $(grep -E '^(ruff)' reqs/requirements-dev-contrib.txt)
           # https://www.flake8rules.com/rules/F401.html
-          flake8 --select F401 great_expectations tests
+          ruff --select F401 great_expectations tests
```

Comment:
@anthonyburdi if you want to have ruff only check for a particular code or category of violations you can use `--select` just like with `flake8`.

So you could do `ruff --select D` to check all the pydocstyle rules
https://github.com/charliermarsh/ruff#pydocstyle-d